app/jobs/resolution_proofing_job.rb

```diff
@@ -16,6 +16,7 @@ class ResolutionProofingJob < ApplicationJob
 
   def perform(
     result_id:,
+    document_capture_session_uuid:,
```

Comment:
Do we need to support this being `nil` for the 50/50 state?

Reply:
Good point! I started to add a spec to resolution_proofing_job_spec to confirm, and it isn't setting document_capture_session_uuid at all (maybe it should...) so that part works fine. And I checked that the underlying test bucket code [returns the default on a nil discriminator](https://github.com/18F/identity-idp/blob/42a5f9e1661a0c95a8a4cec762bcf9187904164f/lib/ab_test_bucket.rb#L14). Doug and I also observed it working this way before the job had access to the uuid (which is why we had to add it).